packages/salesforcedx-apex-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -59,6 +58,8 @@ export interface LaunchRequestArguments
   logFile: string;
   stopOnEntry?: boolean | true;
   trace?: boolean | string;
+  projectBreakpointFilePath?: string;
```

Comment:
Extending this interface to add the file path to the new file containing Apex LSP info & the project path. These used to be included as part of `LineBreakpointEventArgs` but I didn't use that since I would have to port it into `salesforcedx-vscode-apex` to avoid circular dependencies.